README.md

```diff
@@ -2,10 +2,43 @@
 
 The Node.js Foundation Technical Steering Committee is the technical governing body of the Node.js Foundation. It admits and oversees all top-level Projects in the Node.js Foundation. It also elects a representative to the Node.js Foundation Board of Directors.
 
+The TSC is community and contributor driven and is a home for community managed 
+projects and assets. The mission of the Node.js Foundation is to enable and encourage 
+the adoption of Node.js and the TSC is the home of the community driven portion
+of executing that mission.
+
+Since the Foundation and the TSC are intended to support the community the size and scope
+of the work and projects in it will grow to accomodate the needs of the community.
+
```

Comment:
> This statement assumes that growing the size of the size and scope of the work and projects is the only / best way to support the community. It also implies unconditional growth of the scope as a mission of the TSC. 

That's not an assumption I'm trying to make. There's a difference here to the TSC being *reactive* and growing to meet demands of projects in the ecosystem and the TSC being *proactive* and growing "just to grow."

There's plenty the foundation is doing to support the community completely outside of the TSC. I think there's a false impression here that the TSC needs to remove resources from Core or proactively create new resources in order to be a home to projects that are asking for support. I don't believe that is the case, I think that we can admit projects at little to no cost to the current work we're doing. But the pertinent question here how we respond to projects *asking* us for this support. Do we tell them that we have some abstract criteria based on our own personal priorities that limits what kinds of projects we'll allow in or do we simply try to respond to what the community is asking for.

> As we discussed previously, diluting the limited resources is one of the potential drawbacks of such unconditional growth. 

What exactly are these resources? Other than mentorship, which is primarily on my shoulders early on, what exactly are we diluting? What resources are we allocating to these projects that we're in danger of running out of?

> Not objecting to growth in principle but I think that the passage from the what (the mission of the TSC, fostering the Node.js ecosystem) to the how needs to be further elaborated and this statement may fail to capture the complexity of the criteria involved.

Here's the dilemma, how do we figure out what the Node.js ecosystem needs other than responding to what they are asking for? How do we establish the *how* of "how to support the ecosystem" other than to admit those that apply? We are not representative enough of the ecosystem to build objective criteria to what they need and the ecosystems is constantly changing and its needs may change as well.